docs/source/virtual_ecosystem/implementation/hydrology_implementation.md

```diff
@@ -1,29 +1,443 @@
+model which is part of the default Virtual Ecosystem configuration. The key processes
+are illustrated in {numref}`hydrology`.
+
+The processes [within a grid cell](#within-grid-cell-hydrology) are loosely based
+on the LISFLOOD model {cite}`van_der_knijff_lisflood_2010`. The processes
+[across the model grid](#across-grid-hydrology) are loosely based on
+the [pysheds](https://github.com/mdbartos/pysheds) package.
+
+:::{figure} ../../_static/images/hydrology.jpg
+:name: hydrology
+:alt: Hydrology
+:class: bg-primary
+:width: 600px
+
+Hydrology processes in Virtual Ecosystem (click to zoom). Yellow boxes
+represent atmospheric input variables, green box and arrows indicate where water
+enters and leaves the plant model.
+:::
 
 ```{note}
-Some of the features described here are not yet implemented.
+Many of the underlying processes are problematic at a monthly timestep, which is
+currently the only supported update interval. As a short-term work around, the input
+precipitation is randomly distributed over 30 days and input evapotranspiration is
+divided by 30, and the return variables are monthly means or monthly accumulated values.
+```
+
+## Required variables
+
+The tables below show the variables that are required to initialise the hydrology model
+and then update it at each time step.
+
+```{code-cell} ipython3
+---
+tags: [remove-input]
+mystnb:
+  markdown_format: myst
+---
+
+from IPython.display import display_markdown
+from var_generator import generate_variable_table
+
+display_markdown(
+    generate_variable_table(
+        'HydrologyModel', 
+        ['vars_required_for_init', 'vars_required_for_update']
+    ), 
+    raw=True
+)
 ```
 
-## Vertical hydrology components
+The model also requires several parameters that as described in detail in
+{py:class}`~virtual_ecosystem.models.hydrology.constants.HydroConsts`.
+The default values are set for forest ecosystems.
+
+## Within grid cell hydrology
 
 The vertical component of the hydrology model determines the water balance within each
 grid cell. This includes [above ground](../../api/models/hydrology/above_ground.md)
-processes such as rainfall, intercept, and surface runoff out of the grid cell.
+processes such as rainfall, canopy interception, and surface runoff out of the grid cell.
 The [below ground](../../api/models/hydrology/below_ground.md) component considers
 infiltration, bypass flow, percolation (= vertical flow), soil moisture and matric
 potential, horizontal sub-surface flow out of the grid cell, and changes in
 groundwater storage.
-The model is loosely based on the LISFLOOD model {cite}`van_der_knijff_lisflood_2010`.
 
-## Horizontal hydrology components
+### Canopy interception
+
+Canopy interception is estimated using the following storage-based equation after
+{cite:t}`aston_rainfall_1979` and {cite:t}`merriam_note_1960` as implemented in
+{cite:t}`van_der_knijff_lisflood_2010`:
+
+$$Int = S_{max} * [1 - \exp(\frac{-k*R*\delta t}{S_{max}})]$$
+
```

Comment:
```suggestion
$$\textrm{Int} = S_{max}  \left[1 - \exp\left(\frac{-k*R*\delta t}{S_{max}}\right)\right]$$
```

The `\left[` etc. commands allow the brackets to grow vertically to fit around the equation.

Also - here and in other equations - I think we can drop a load of the `*` multiplication signs. Where it does help readability, it is canonical to use `\cdot` (or `\times`).